lib/WebService/Stripe.pm

```diff
@@ -68,6 +68,16 @@ method create_charge(HashRef $data, :$headers) {
     return $self->post( "/v1/charges", $data, headers => $headers );
 }
 
+method get_charge(Str $charge_id, :$expand, :$headers) {
```

Comment:
Every GET request supports the `expand` parameter, so we should probably find a nicer way of handling this.